.github/workflows/sonar_analysis.yml

```
@@ -0,0 +1,75 @@
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Wait for SonarQube to Start
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:2:1: i appears unused. Verify use (or export if used externally)

(shellcheck)